eqcorrscan/core/subspace.py

```diff
@@ -586,6 +582,41 @@ def _detect(detector, st, threshold, trig_int, moveout=0, min_trig=0,
     return detections
 
 
+def _do_ffts(detector, stream, Nc):
```

Comment:
Can we have a doc-string for this please? I like to have them even for internal functions so we know what's going on (I probably don't have them for everything though!)